n3fit/src/n3fit/scripts/n3fit_exec.py

```diff
@@ -133,7 +133,9 @@ def from_yaml(cls, o, *args, **kwargs):
 
         N3FIT_FIXED_CONFIG['actions_'].append(namespace + "performfit")
 
-        if file_content["fitting"].get("savepseudodata"):
+        if file_content["fitting"].get("savepseudodata", True):
+            if file_content["fitting"]["savepseudodata"] != True:
+                raise TypeError("fitting::savepseudodata is neither True nor False")
```

Comment:
Yes, I'm happy enough with that.

Also a nice opportunity to add a walrus...
```suggestion
        if fps := file_content["fitting"].get("savepseudodata", True):
            if fps != True:
                raise TypeError(f"fitting::savepseudodata is neither True nor False ({fps})")
```
(feel free to ignore this whilst sighing :P)

Reply:
Fair enough, we don't support 3.7 anyway